src/ontology/components/dogbreeds.tsv

```diff
@@ -425,7 +425,7 @@ VBO:0200337	Chien-gris (Dog)	Chien-gris	Dog				NCBITaxon:9615						Chien-gris	ht
-VBO:0200342	Chilean Terrier (Dog)	Chilean Terrier	Dog				NCBITaxon:9615						Chilean Terrier	http://purl.obolibrary.org/obo/vbo#most_common_name	https://en.wikipedia.org/wiki/Chilean_Terrier																																																																																															https://en.wikipedia.org/wiki/Chilean_Terrier		https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153|https://orcid.org/0000-0002-5520-6597|https://orcid.org/0000-0002-9178-3965|https://orcid.org/0000-0001-7688-1770																								
+VBO:0200342	Chilean Terrier (Dog)	Chilean Terrier	Dog			https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/122	NCBITaxon:9615						Chilean Terrier	http://purl.obolibrary.org/obo/vbo#most_common_name	https://en.wikipedia.org/wiki/Chilean_Terrier	PMID:37778752																																																																																														https://en.wikipedia.org/wiki/Chilean_Terrier		https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153|https://orcid.org/0000-0002-5520-6597|https://orcid.org/0000-0002-9178-3965|https://orcid.org/0000-0001-7688-1770																								

```

Comment:
@katiermullen 
The PMID was added for the most common name (and not as a "source" for the breed). If this is what was expected, then it is ready to merge. If the expectation was to add as a source, then it should be updated before merging

Reply:
Oh yes. I will fix this.